kmip/core/factories/secrets.py

```diff
@@ -68,7 +69,7 @@ def create(self, secret_type, value=None):
             SymmetricKey(...)
         """
         if secret_type is ObjectType.CERTIFICATE:
-            return self._create_certificate(value)
+            return self._create_certificate()
```

Comment:
I plan to pass you a value for this just like with public and private keys. The value will be a DER encoded X.509 certificate.